index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <title>
```

Comment:
title should be inside the <head></head>

Reply:
Done